src/client/components/ChangesetDelta/ChangesetDelta.js

```diff
@@ -1,22 +1,42 @@
 /* @flow */
 import React from 'react'
+import PieChart from 'react-simple-pie-chart'
```

Comment:
Pie charts are generally bad as they are hard for humans to perceive relative spatial significance. I would think a simple bar chart would fit a lot better.